.packit.yml

```diff
@@ -0,0 +1,48 @@
+upstream_package_name: rhc
+downstream_package_name: rhc
+specfile_path: rhc.spec
+
+srpm_build_deps:
+  - gawk
+  - git-core
+  - golang
+  - make
+  - 'pkgconfig(dbus-1)'
+  - 'pkgconfig(systemd)'
+  - rpm-build
+
+actions:
+  post-upstream-clone:
+    - bash -c 'make spec dist'
+  get-current-version:
+    - awk '/^Version:/ {print $2;}' ./rhc.spec
+  create-archive:
+    - bash -c 'echo rhc-*.tar.*'
+  fix-spec-file:
+    - echo 'nothing to fix'
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    additional_repos:
+      - "copr://@yggdrasil/latest"
+    targets:
+      - centos-stream-8
+      - centos-stream-9
+      - fedora-all
+      - rhel-8
+      - rhel-9
+
+  - job: copr_build
+    trigger: commit
+    additional_repos:
+      - "copr://@yggdrasil/latest"
```

Comment:
Does this additional repo need to be here when you're already building into that same repo?

Reply:
For some reason we have `BuildRequires: yggdrasil >= 0.4` in `rhc.spec.in` file. Is it really necessary for upstream repository?

Reply:
Can we try to fix this in another PR? It seems that yggdrasil is not needed for building `rhc`? I don't want to mix too many things together.

Reply:
Yea, that's fair. Let's clean this up in a followup PR.